### ISSUE TITLE
errcheck no longer works

### ISSUE THREAD

**Issue by matrixik** [alecthomas/gometalinter#6]
Hi,

I'm not sure if I did something wrong or `gometalinter` do not run `errcheck` on files

My output from example folder:
```
gometalinter .
stutter.go:22::error: Repeating defer a.Close() inside function duplicateDefer
stutter.go:9::warning: unused global variable unusedGlobal
stutter.go:13::warning: unused struct field MyStruct.Unused
stutter.go:12:6:warning: exported type MyStruct should have comment or be unexported
stutter.go:16:6:warning: exported type PublicUndocumented should have comment or be unexported
stutter.go:29::error: unreachable code
stutter.go:26::error: missing argument for Printf("%d"): format reads arg 1, have only 0 args
```
It should show error/warning also for line 27.
Can be related to https://github.com/kisielk/errcheck/issues/45

**Comment by alecthomas** [comment]
If you do a `gometalinter --update --install` you will get my forked version of errcheck that should work.

**Comment by matrixik** [comment]
It's working now. Thank you!
